Scripts/update.sh

```diff
@@ -32,6 +32,7 @@ for REPO in $REPOS; do
 			git checkout master
 			git reset --hard origin/master
 			~/bin/BuildGit "$REPO" "$MODROOT" "$MODINFO" "$PACKAGES/$TAG" "$REMOTE" "$TAG"
+			git submodule update --recursive --remote
```

Comment:
1. --init should be added
2. should be run before BuildGit

Reply:
oh, oops

Reply:
thats true ... sorry my fault